src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
```

Comment:
The link to the stylesheet is pointing to an .scss file. Typically, it should link directly to a CSS file in an HTML document. Ensure that the SCSS file is compiled to CSS before using it in HTML. Alternatively, you can change the link to point directly to a CSS file.